### PR TITLE
added microdata to hugo footer.html

### PR DESCRIPTION
**Pull request by Nayananga**
I`m not sure whether this is the correct way of adding microdata. Please let know. And if this data okay, Ill find a way to incorporate them antora camel themes  as well

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ documentation
 .metadata
 .pnp
 .pnp.js
+.settings/org.eclipse.core.resources.prefs
+.settings/org.eclipse.m2e.core.prefs
```
Comment (zregvart) [inline .gitignore]: You can put whole `.settings` in `.gitignore` I'm sure other files will pop up in it.

File: content/community/support.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Support"
 ---
 
-## Getting help
+## Getting help
```
Comment (zregvart) [inline content/community/support.md]: :+1: good catch

File: layouts/partials/footer.html [hunk 3]
```diff
@@ -47,5 +47,13 @@ <h5>About</h5>
...
     <script src="{{ "/_/js/site.js" | relURL }}"></script>
     <script src="{{ "/_/js/vendor/highlight.js" | relURL }}"></script>
+    <script type="application/ld+json">
+        {
+        "@context": "http://schema.org/",
+        "@type": "Organization",
+        "url": "https://camel.apache.org/staging/",
+        "logo": "https://camel.apache.org/staging/_/img/logo-d.svg"
+        }
+      </script>
```
Comment (zregvart) [inline layouts/partials/footer.html]:
```suggestion
    </script>
```

To make it align with the opening `<script>` tag.

File: layouts/partials/footer.html [hunk 4]
```diff
@@ -47,5 +47,13 @@ <h5>About</h5>
     </footer>
     <script src="{{ "/_/js/site.js" | relURL }}"></script>
     <script src="{{ "/_/js/vendor/highlight.js" | relURL }}"></script>
+    <script type="application/ld+json">
+        {
+        "@context": "http://schema.org/",
+        "@type": "Organization",
+        "url": "https://camel.apache.org/staging/",
+        "logo": "https://camel.apache.org/staging/_/img/logo-d.svg"
```
Comment (zregvart) [inline layouts/partials/footer.html]: These URLs need to come from the [Hugo variables](https://gohugo.io/variables/site/), we won't be publishing to `/staging` for long, so best make that configurable. Probably best to use `.Site.BaseURL` here that's configured from the config.toml here:

https://github.com/apache/camel-website/blob/85632c2c1059b528d688d5e2bf1616b06722e9ba/config.toml#L1

Comment (Nayananga) [reply layouts/partials/footer.html]: Hello Mr.Zoran, Can you check now?

**Pull request merged**